### PR TITLE
Feature/graphql

### PR DESCRIPTION
**Pull request by Kerraan**

### CODE REVIEW

File: apps/team-awesome-web/pages/index.tsx [hunk 1]
```diff
@@ -1,72 +1,74 @@
...
-    if (!window.confirm("are you sure")) {
-      return;
-    }
-    const result = await deleteTimeEntry(timeEntryToDelete);
-    if (result instanceof Error) {
-      setErrorWarning("Time entries could not be deleted");
-      return;
-    }
+  const handleDeleteEntry = async (id: string) => {
+    // console.log(id);
```
Comment (nathaliebru) [inline apps/team-awesome-web/pages/index.tsx:68]: Remove this comment

File: apps/team-awesome-web/src/components/Form/team-member-form/TeamMemberForm.tsx [hunk 2]
```diff
@@ -4,10 +4,9 @@ import { Button } from "../../button";
 import { Input } from "../input/Input";
 import * as Styled from "./TeamMemberForm.styled";
 import { StoreContext } from "../../store-context";
-import { TeamMember } from "../../../types";
 
 export const defaultMember = {
-  id: 0,
+  id: "0",
```
Comment (nathaliebru) [inline apps/team-awesome-web/src/components/Form/team-member-form/TeamMemberForm.tsx:9]: Just curious, why did you change this `id` to a string?

File: apps/team-awesome-web/pages/_app.tsx [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
 function MyApp({ Component, pageProps }: AppProps) {
   return (
     <>
-      <ThemeProvider theme={theme}>
-        <GlobalStyle />
-        <StoreProvider>
-          <Component {...pageProps} />
-        </StoreProvider>
-      </ThemeProvider>
+      <ApolloProvider client={client}>
```
Comment (b3rgt0p) [inline apps/team-awesome-web/pages/_app.tsx:13]: @nathaliebru Is this the correct nesting for `<ApolloProvider>`?

**Review by nathaliebru — COMMENTED** [review]
Nicely rewritten to graphQL!🚀 Just one comment and a question.

File: apps/team-awesome-web/pages/index.tsx [hunk 4]
```diff
@@ -1,72 +1,74 @@
...
-  const response = await getTimeEntries();
-
-  if (response instanceof Error) {
-    return {
-      props: {
-        errorMessage: "This is an error",
-      },
-    };
-  }
+  const { loading, error, data } = await apolloClient.query({
```
Comment (b3rgt0p) [inline apps/team-awesome-web/pages/index.tsx:25]: Nice!

File: apps/team-awesome-web/pages/index.tsx [hunk 5]
```diff
@@ -1,72 +1,74 @@
...
-    if (result instanceof Error) {
-      setErrorWarning("Time entries could not be deleted");
-      return;
-    }
+  const handleDeleteEntry = async (id: string) => {
+    // console.log(id);
+    await removeTimeEntry({ variables: { id } });
 
-    const afterDelete = timeEntries.filter((entry) => entry.id !== timeEntryToDelete.id);
+    const afterDelete = timeEntries.filter((entry) => entry.id !== id);
```
Comment (b3rgt0p) [inline apps/team-awesome-web/pages/index.tsx:71]: Without spreading you're filtering `timeEntries` here in-place; is this preferred [behavior](https://developer.mozilla.org/en-US/docs/Glossary/Shallow_copy)?

File: apps/team-awesome-web/src/components/Form/team-member-form/TeamMemberForm.tsx (hunk 2 above)
Comment (Kerraan) [reply apps/team-awesome-web/src/components/Form/team-member-form/TeamMemberForm.tsx:9]: I changed it to string because in graphQL GET_TIME_ENTRIES i also made it a string yesterday with Fabiano and Herman. When i pushed it this morning a type error came up saying it cannot compare string as a number or something when using the default entry because i re-typed it as a string. Could be number either way.

**Review by b3rgundy — APPROVED** [review]
Looks shippable🚢

**Review by nathaliebru — APPROVED** [review]
NICE!!!
